app/pages/api/ask.ts

```diff
@@ -104,11 +113,13 @@ export default async function handler(
     res.status(405).json(error({ errors: ["Method not allowed"] }));
     return;
   }
-
-  const shouldStream = USE_STREAMING && Boolean(req.query.stream);
+  const shouldUseLlm = req.query.withLlm === "true";
+  const shouldStream = !shouldUseLlm
+    ? false
+    : USE_STREAMING && req.query.stream === "true";
```

Comment:
```suggestion
  const shouldStream = shouldUseLlm && USE_STREAMING && req.query.stream === "true";
```
nit: can avoid negation & ternary with another &&